python_ta/reporters/templates/stylesheet.css

```diff
@@ -134,3 +134,25 @@ body {
     text-align: center;
     padding-top: 10px;
 }
+
+#.hidden {
```

Comment:
try to avoid using `id` selectors.

remember, id selected with `#`, class selected with `.`